mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
+	FIELD field_25136 INSTANCES Ljava/util/Map;
+	FIELD field_25137 registryId Lnet/minecraft/class_2960;
+	FIELD field_25138 valueId Lnet/minecraft/class_2960;
+	METHOD <init> (Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)V
+		ARG 1 registryId
+		ARG 2 valueId
+	METHOD method_29177 getValueId ()Lnet/minecraft/class_2960;
+	METHOD method_29179 getOrCreate (Lnet/minecraft/class_5321;Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
+		ARG 0 registryId
+		ARG 1 valueId
+	METHOD method_29180 getOrCreateRegistryKey (Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
```

Comment:
```suggestion
	METHOD method_29180 getOrCreate (Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
```
The rename is probably fine as the descriptor differs

Reply:
This is specifically an overload where a registry key for Registry.ROOT is created (i.e. an ID for a registry). I'll rename it to "root key"